packages/Toggle/src/toggle.scss

```
@@ -1,142 +1,225 @@
 @use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
 @use '~@igloo-ui/tokens/dist/fonts';
 
+@import url('~@hopper-ui/tokens/fonts.css');
+@import url('~@hopper-ui/tokens/tokens.css');
+
 :root {
-  /* Default */
-  --ids-toggle-font-family: #{tokens.$primary-font-family};
-  --ids-toggle-font-size: #{tokens.$font-size-4};
-  --ids-toggle-text-padding: #{tokens.$space-2};
-  --ids-toggle-width: #{tokens.$space-6};
-  --ids-toggle-height: 2rem;
-  --ids-toggle-background: #{tokens.$grey-300};
-  --ids-toggle-transition: background-color 0.3s ease-in-out;
-  --ids-toggle-focus: #{tokens.$focus};
-  --ids-toggle-padding: #{tokens.$space-1};
-  --ids-toggle-font-size-description: #{tokens.$font-size-3};
-  --ids-toggle-text-description: #{tokens.$grey-600};
-
-  /* switch */
-  --ids-toggle-width-switch: #{tokens.$space-3};
-  --ids-toggle-height-switch: #{tokens.$space-3};
-  --ids-toggle-background-switch: #{tokens.$samoyed};
-  --ids-toggle-shadow-switch: 0 .1rem .3rem 0 rgb(0 0 0 / 25%);
-
-  /* checked */
-  --ids-toggle-background-checked: #{tokens.$electric-blue-500};
-  --ids-toggle-position-checked:
-    calc(
-      var(--ids-toggle-width) - var(--ids-toggle-width-switch) - 0.4rem
-    );
-
-  /* disabled */
-  --ids-toggle-background-disabled: #{tokens.$grey-200};
-  --ids-toggle-background-checked-disabled: #{tokens.$electric-blue-300};
+    /* Default */
+    --ids-toggle-font-family: #{tokens.$primary-font-family};
+    --ids-toggle-font-size: #{tokens.$font-size-4};
+    --ids-toggle-line-height: normal;
+    --ids-toggle-text-padding: #{tokens.$space-2};
+    --ids-toggle-font-weight: 400;
```

I know it might not matter eventually but this should be:
```suggestion
    --ids-toggle-font-weight: #{tokens.$font-weight-regular};
```